diff -Nruw codec.orig/amr-code/makefile codec/amr-code/makefile
--- codec.orig/amr-code/makefile	2016-12-18 15:57:20.737366601 +0100
+++ codec/amr-code/makefile	2016-12-18 16:00:02.166031929 +0100
@@ -21,7 +21,7 @@
 
 CC=gcc
 
-CFLAGS	=	-g -I. -Wall -m32
+CFLAGS	=	-g -I. -Wall -m32 -static
 
 SRCS1		=	ccoder.c	ccod_tet.c	init_params.c	sub_cc.c \
 			tetra_op.c
diff -Nruw codec.orig/c-code/makefile codec/c-code/makefile
--- codec.orig/c-code/makefile	2016-12-18 15:57:20.737366601 +0100
+++ codec/c-code/makefile	2016-12-18 15:59:55.146002990 +0100
@@ -23,7 +23,7 @@
 
 CC=gcc
 
-CFLAGS	=	-I. -Wall -m32
+CFLAGS	=	-I. -Wall -m32 -static
 
 SRCS1		=	scoder.c		scod_tet.c		sub_sc_d.c \
 			sub_dsp.c		fbas_tet.c		fexp_tet.c \
Binary files codec.orig/c-code/scoder and codec/c-code/scoder differ
Binary files codec.orig/c-code/sdecoder and codec/c-code/sdecoder differ
